exonum/src/helpers/types.rs

```diff
@@ -51,7 +51,7 @@ impl Height {
     /// let next_height = height.next();
     /// assert_eq!(11, next_height.0);
     /// ```
-    pub fn next(&self) -> Self {
+    pub fn next(self) -> Self {
```

Comment:
And what do you think about this changes. 
It catched by trivially_copy_pass_by_ref  with error: 'this argument is passed by reference, but would be more efficient if passed by value'

Reply:
It is common to pass copy-types by value.